src/templates/button.mustache

```diff
@@ -1,5 +1,5 @@
 {{#buttonUrl}}
-    <a href="{{{ buttonUrl }}}" target="_blank">{{{ buttonLabel }}}</a>
+    <a href="{{{ buttonUrl }}}" target="_blank" {{#buttonUrl}}aria-label="{{ ariaLabel }}" role="button"{{/buttonUrl}}>{{{ buttonLabel }}}</a>
```

Comment:
I'm confused here. If there is a buttonUrl, then I think this should be a link, not a role=button. Is that right? Unless I'm missing some negator in the code here.

It looks like I'll get <a href="something.com" target="_blank" aria-label="label" role="button">other label</a>
or <a href="" target="_blank">other label</a>

That first one looks like a link that wouldn't need the role and would only need the label if the other label wasn't properly descriptive. That second example looks like a button that needs the aria attributes.

Maybe #buttonUrl was supposed to be #ariaLabel?